mantidproject/mantid
# SumOverlappingTubes produce histogram data

**Description of work.**

This PR modifies `SumOverlappingTubes` algorithm to produce histogram data and not to split the counts by default. This is the desired behaviour. Tests are adjusted correspondingly.

**To test:**

```
Load(Filename='538998.nxs', OutputWorkspace='ws')
CropToComponent(InputWorkspace='ws', OutputWorkspace='tube1', ComponentNames='tube_1')
SumOverlappingTubes(InputWorkspaces='tube1', OutputWorkspace='sum', OutputType='2DTubes', ScatteringAngleBinning=0.052)
```
Make sure `sum` is a histogram workspace.
Plot 2D and check it has no empty bin (column) around the middle of x-axis.
File is attached, cut off `.zip`.
[538998.nxs.zip](https://github.com/mantidproject/mantid/files/2175713/538998.nxs.zip)




Fixes #22792 . 

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
